examples/workflows/child_workflow_terminated_workflow.rb

```diff
@@ -0,0 +1,20 @@
+require 'workflows/simple_timer_workflow'
+require 'activities/terminate_workflow_activity'
+
+class ChildWorkflowTerminatedWorkflow < Temporal::Workflow
+  def execute
+    # start a child workflow that executes for 60 seconds, then attempts to try and terminate that workflow
+    result = SimpleTimerWorkflow.execute(60)
+    child_workflow_execution = result.child_workflow_execution_future.get
+    TerminateWorkflowActivity.execute!(
+      'ruby-samples',
+      child_workflow_execution.workflow_id,
+      child_workflow_execution.run_id
+    )
+
+    # check that the result is now 'failed'
+    {
+      child_workflow_terminated: result.failed? # terminated is represented as failed? with the Terminated Error
```

Comment:
It would be good to return the actual error as well here, otherwise the spec won't fail if the child workflow fails for any other reason

Reply:
agreed! added this to both places